src/script/cpp_api/s_security.cpp

```diff
@@ -963,3 +1100,12 @@ int ScriptApiSecurity::sl_os_setlocale(lua_State *L)
 	lua_call(L, cat ? 2 : 1, 1);
 	return 1;
 }
+
+
+int ScriptApiSecurity::sl_os_clock(lua_State *L)
+{
+	auto t = clock();
+	t = t - t % (SSCSM_CLOCK_RESOLUTION_US * CLOCKS_PER_SEC / 1'000'000);
```

Comment:
```suggestion
	t = t - t % (SSCSM_CLOCK_RESOLUTION_US * CLOCKS_PER_SEC / 1000000);
```

these things are legal but too often not supported well. `/ 1000 / 1000` or similar if you think the six zeros are unreadable.

Reply:
> Or do you mean that there are tools that do not support it?

yes. i know that micro's syntax highlighting doesn't. i remember there being more, but i tried a few editors now and maybe this quirk really is limited to just micro (which generally gets a few syntax highlighting things wrong) 🤔 

i'll resolve this unless something else pops up.